forge/test/operators/utils/compat.py

```diff
@@ -22,12 +24,198 @@ class TestDevice:
     pass
 
 
+class TestTensorsUtils:
+
+    dev_data_format_to_dtype = {
+        # forge.DataFormat.Bfp2: torch.float16,
+        # forge.DataFormat.Bfp2: torch.bfloat16,
+        forge.DataFormat.Bfp2: torch.float32,
+        forge.DataFormat.Bfp2_b: torch.float32,
+        # forge.DataFormat.Bfp2_b: torch.bool,  # mapped
+        forge.DataFormat.Bfp4: torch.float32,
+        forge.DataFormat.Bfp4_b: torch.float32,
+        forge.DataFormat.Bfp8: torch.float32,
+        forge.DataFormat.Bfp8_b: torch.float32,
+        # forge.DataFormat.Float16: torch.float16,  # fatal python error
+        forge.DataFormat.Float16: torch.float32,  # mapped
+        forge.DataFormat.Float16_b: torch.float32,  # mapped
+        forge.DataFormat.Float32: torch.float32,  # mapped
+        # forge.DataFormat.Lf8: torch.float16,  # fatal python error
+        # forge.DataFormat.Lf8: torch.bfloat16,  # pcc failed
+        forge.DataFormat.Lf8: torch.float32,
+        # forge.DataFormat.RawUInt8: torch.int8,  # unsupported
+        # forge.DataFormat.RawUInt8: torch.uint8,  # Unsupported torch dtype
+        forge.DataFormat.RawUInt8: torch.int32,
```

Comment:
I would add some comment here to explain why we have this mapping, and that we won't have precission loss because we tailor our domen of generated values...